modules/rules_java/7.0.2/MODULE.bazel

```diff
@@ -0,0 +1,78 @@
+module(
+    name = "rules_java",
+    version = "7.0.2",
+    bazel_compatibility = [">=6.4.0"],
```

Comment:
The default is `compatibility_level = 0`, so this needs `compatibility_level = 1` to be interoperable with previous versions.

Reply:
@hvadehra

Reply:
Aargh

Reply:
Third time's the charm: https://github.com/bazelbuild/bazel-central-registry/pull/1025